spec/new_relic_spec.rb

```diff
@@ -26,12 +38,30 @@
   end
 
 
+  context 'with correct setup' do
+    include_examples 'loads'
+  end
+
   context 'when NEW_RELIC_LICENSE_KEY is missing' do
     before do
       app_helper.comment_lines app_path.join('.env'), /NEW_RELIC_LICENSE_KEY/
     end
   
-    include_examples 'abort early', /NEW_RELIC_LICENSE_KEY is required/
+    context 'in the test environment' do
+      let(:app_env) { 'test' }
+
+      include_examples 'does not load'
```

Comment:
Ah OK so it's not specifically related to the code here, just part of what happens in the NR gem itself? In that case I think it's fine to have it different, but maybe add a comment to the tests as to why it's different.

Do we actually want/need it to load in dev anyway though?